<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PpimPpie</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../PYtofile/ppimppie_icon.ico</normaloff>../PYtofile/ppimppie_icon.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f0f0;
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>궁서체</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: white;
    color: blue;
	border: 1px solid blue;
}
</string>
   </property>
   <property name="text">
    <string>삠삐</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>궁서</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: white;  /* 배경색 흰색 */
        color: black;             /* 글자색 검정 */
    }
    QPushButton:hover {
        background-color: #f9f9f9;  /* 마우스 올릴 때 연한 회색 */
    }
    QPushButton:pressed {
        background-color: #e6e6e6;  /* 눌렀을 때 더 진한 회색 */
    }</string>
   </property>
   <property name="text">
    <string>시작하기</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    background:white;       /* 배경색 */
}
</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
